packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/index.md

```diff
@@ -6,7 +6,7 @@ If you are building a modern app or API, you likely want to know if your end use
 
 Once the signed tokens are issued to the end users they can be passed to your application, which must validate them. There are two ways to verify a token: locally, or remotely with Okta. The token has been signed with a JSON Web Key (JWK) using the RS256 algorithm. To validate the signature, Okta provides your application with a public key that can be used.
```

Comment:
As per Microsoft Manual of Style, suggest we change this instance of "Once" to "After"